recipes/aim/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "aim" %}
+{% set version = "2.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/aimhubio/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: af4f06e7ecc3b30537370e732f172670588de580eaa073c7eeed248dec63f248
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - googlesearch
+    - protobuf
+    - anytree
+    - pyrser
+    - psutil
```

Comment:
These are likely only run dependencies

Reply:
Sorry, looks like they are actually needed as the package imports itself as build time: https://github.com/aimhubio/aim/blob/7d31a0feeb2d958cbaf5ed34e410737168a28c3d/setup.py#L7

Reply:
just retested, I received build failures for each google/protobuf, anytree, pyrser and psutil when I remove them from host